composer.json

```diff
@@ -45,12 +53,12 @@
             "vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-html tests/html-report"
         ],
         "phpcs": [
-            "vendor/bin/php-cs-fixer fix --config-file=.php_cs --verbose --diff --dry-run"
+            "vendor/bin/php-cs-fixer fix --config=.php_cs.dist --verbose --diff --dry-run"
```

Comment:
No need for dist. It doesn't make sense in this context as you're not ever overwriting ours.